server/tests/api/userTests.js

```diff
@@ -0,0 +1,72 @@
+import chai from 'chai';
+import request from 'supertest';
+import 'babel-polyfill';
+
+import User from './../../models/User';
+import users from './../testData/users';
+import app from './../../app';
+
+const { expect } = chai;
+
+describe('UserController', () => {
+  before(async () => {
+    await User.remove({});
+  });
+
+  describe('When a user signs up', () => {
+    it('Then it should return the user details and authentication token', async () => {
```

Comment:
Consider saying `it('returns the user details and authentication token')`. So that when we read the whole thing it  looks like this => `When a user signs up, it returns the user details and authentication token`

Reply:
Okay. Thanks. Will do that